Assets/HealthUI.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthUI : MonoBehaviour
+{
+    [SerializeField] private GameObject[] dragonHealthUI;
+    [SerializeField] private GameObject[] knightHealthUI;
+    [SerializeField] private int dragonHealth;
+    [SerializeField] private int knightHealth;
+    // Start is called before the first frame update
+    void Start()
+    {
+        foreach (GameObject life in dragonHealthUI)
+        {
+            life.SetActive(true);
+        }
+        foreach (GameObject life in knightHealthUI)
+        {
+            life.SetActive(true);
+        }
+    }
+
+    public int DragonHealth { get => dragonHealth; }
+
+    public int KnightHealth { get => knightHealth; }
```

Comment:
Since these are public properties, they should go below the serialized private fields above, before the `Start` method.